gatsby-node.js

```diff
@@ -20,4 +20,6 @@ exports.createPages = async ({ actions: { createPage }, graphql }) => {
       context: {},
     });
   });
+
+  return Promise.resolve();
```

Comment:
This is redundant

Reply:
Realy? the function is async and in if statement it returns a promise.reject() but in other case it does not returns anything.

Reply:
async function always returns a promise.
`Promise.reject()` is used to basically throw an error (that's what I should've done in a first place 🤔 )